adapter/src/dbt/adapters/fal_experimental/utils/environments.py

```diff
@@ -110,11 +88,13 @@ def load_environments(base_dir: str) -> Dict[str, BaseEnvironment]:
 def create_environment(name: str, kind: str, config: Dict[str, Any]):
     from isolate.backends.virtualenv import VirtualPythonEnvironment
     from isolate.backends.conda import CondaEnvironment
+    from isolate.backends.local import LocalPythonEnvironment
+
 
     REGISTERED_ENVIRONMENTS: Dict[str, BaseEnvironment] = {
         "conda": CondaEnvironment,
         "venv": VirtualPythonEnvironment,
-        "local": LocalEnvironment
+        "local": LocalPythonEnvironment
```

Comment:
OK, so I just realized that this means we have to call `create_environment` with `name=local, kind=local`. Which seems weird.

Maybe we can remove `local` form the dict and keep the `if` statement we removed... But **do** use the isolate local environment